src/main/java/Duke.java

```diff
@@ -1,10 +1,225 @@
+import Exceptions.*;
+//import Tasks.Deadline;
+//import Tasks.Event;
+//import Tasks.Task;
+//import Tasks.ToDo;
+import Tasks.*;
+
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.util.Arrays;
+
 public class Duke {
-    public static void main(String[] args) {
+    public static final int DEADLINE_STRING_LENGTH = 9;     // "deadline "
+    public static final int BY_SEPARATOR_LENGTH = 4;        // "/by  "
+    public static final int EVENT_STRING_LENGTH = 6;        // "event "
+    public static final int AT_SEPARATOR_LENGTH = 4;        // "/at  "
+    public static ArrayList<Task> list = new ArrayList<Task>();
+
+    public static void printList(ArrayList<Task> list) {
+            System.out.println("        ____________________________________________");
+            for (int i = 1; i <= list.size(); i += 1) {
+                String taskType = list.get(i-1).getTaskType();
+                String markDoneStatus = list.get(i-1).isDone()?"[X]":"[ ]";
```

Comment:
Consider adding space between operators.
```suggestion
                String taskType = list.get(i - 1).getTaskType();
                String markDoneStatus = list.get(i - 1).isDone() ? "[X]" : "[ ]";
```
> Operators should be surrounded by a space character.

https://se-education.org/guides/conventions/java/index.html#layout